### DISCUSSION TITLE
Status of Implementation

### DISCUSSION THREAD

**Discussion by uncracked-concrete** [Dlubal-Software/Dlubal_CSharp_Client#32] [Q&A] [answered]
I noticed that the python library seems to be much more complete at the moment. The C# libraries miss some critical commands like "connect_members" in RSTAB for example. Is more going to be added in the future or are you focusing on the python library?

We cant move to RSTAB9 / RFEM6 as we use the old COM interface extensively and would like to upgrade to the SOAP API. Switching to python would be a lot more work...

**Comment by Rebecca-Coding21** [comment, marked as answer]
Hi @uncracked-concrete,
we continuously keep expanding the library and adding more functions. The C#-library will be updated as well as the Python library,

**Comment by jarabroz** [comment]
Hi @uncracked-concrete ,

C# library is automatically generated based on the underlying WSDL. It means it is up to date with the released application.  Python library is being manually developed. That means C# library is more ahead than Python library. 
Which functionality you are missing in C# compare to Python library - which WebService operations?

**Reply by uncracked-concrete** [reply]
Thanks for replying. That is helpful. I mean for example "Tools/centreOfGravityAndObjectInfo" is available in the Python library but I can't find it in the auto-generated C# library. It's not that we specifically need that command, I was just wondering why the difference...
